test/test_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+require 'minitest/autorun'
+require 'vcr'
+require 'webmock'
+require 'minitest/around/spec'
```

Comment:
you'll also want to require 'pry' here, so that it's always available when interacting with tests.